.github/workflows/CI.yaml

```diff
@@ -0,0 +1,72 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Validate Gradle Wrapper
+        uses: gradle/wrapper-validation-action@v1
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: 11
+
+      - name: Setup Gradle
+        uses: gradle/gradle-build-action@v2
+
+      - name: Run unit tests for the sandboxDebug build
+        run: ./gradlew testSandboxDebugUnitTest --stacktrace
+
+      - name: Build all build variants
+        run: ./gradlew assemble --stacktrace
+
+      - name: Upload App to BrowserStack
+        id: uploadApp
+        uses: fjogeleit/http-request-action@v1
```

Comment:
Can you just use cURL instead of this guy's open source action? Definitely not the kind of dependency we want to rely on for security reasons